src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -127,25 +127,51 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                     </div>
                     <jhi-programming-exercise-lifecycle id="timeline" [exercise]="programmingExercise" [isExamMode]="isExamMode"> </jhi-programming-exercise-lifecycle>
                 </div>
-                <div class="form-group">
-                    <label class="label-narrow" jhiTranslate="artemisApp.exercise.maxScore" for="field_maxScore">Max Score</label>
-                    <input
-                        required
-                        type="number"
-                        [pattern]="maxScorePattern"
-                        class="form-control"
-                        min="1"
-                        max="9999"
-                        name="maxScore"
-                        id="field_maxScore"
-                        [(ngModel)]="programmingExercise.maxScore"
-                        #maxScore="ngModel"
-                    />
-                    <ng-container *ngFor="let e of maxScore.errors! | keyvalue | removekeys: ['required']">
-                        <div *ngIf="maxScore.invalid && (maxScore.dirty || maxScore.touched)" class="alert alert-danger">
-                            <div [jhiTranslate]="'artemisApp.exercise.form.maxScore' + '.' + e.key"></div>
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group-narrow">
+                            <label class="label-narrow" jhiTranslate="artemisApp.exercise.points" for="field_points">Regular Points</label>
+                            <input
+                                required
+                                type="number"
+                                [pattern]="maxScorePattern"
+                                class="form-control"
+                                min="1"
+                                max="9999"
+                                name="points"
+                                id="field_points"
+                                [(ngModel)]="programmingExercise.maxScore"
+                                #maxScore="ngModel"
+                            />
+                            <ng-container *ngFor="let e of maxScore.errors! | keyvalue | removekeys: ['required']">
+                                <div *ngIf="maxScore.invalid && (maxScore.dirty || maxScore.touched)" class="alert alert-danger">
+                                    <div [jhiTranslate]="'artemisApp.exercise.form.maxScore' + '.' + e.key"></div>
```

Comment:
This needs to be points now.
```suggestion
                                    <div [jhiTranslate]="'artemisApp.exercise.form.points' + '.' + e.key"></div>
```